app/src/main/res/drawable/splash_image.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <layer-list xmlns:android="http://schemas.android.com/apk/res/android">
-  <item android:drawable="@color/color_palette_primary_color" />
+  <item android:drawable="@color/color_palette_splash_activity_background_color" />
```

Comment:
```suggestion
  <item android:drawable="@color/component_color_splash_activity_background_color" />
```

Component colors allow the app to adjust to both light and dark modes, and are located in component_colors.xml. This [link](https://github.com/oppia/oppia-android/wiki/Dark-Mode) contains more information about dark mode support at Oppia.

Reply:
Hey @adhiamboperes, thanks for the suggestion! I’m updating the splash screen background to use the component color from `component_colors.xml` so it adapts to both light and dark themes. PTAL once the changes are up.